modin/core/storage_formats/pandas/parsers.py

```diff
@@ -804,6 +827,8 @@ def parse(files_for_parser, engine, **kwargs):
         if isinstance(files_for_parser, (str, os.PathLike)):
             return pandas.read_parquet(files_for_parser, engine=engine, **kwargs)
 
+        to_pandas_kwargs = PandasParser.get_types_mapper(kwargs["dtype_backend"])
```

Comment:
```suggestion
        to_pandas_kwargs = PandasParser.get_types_mapper(kwargs["dtype_backend"])
        del kwargs["dtype_backend"]
```
Is this needed? Why?

Reply:
this is not needed for `read_parquet` because `kwargs` is not passed directly to `_read_row_group_chunk`.